### PR TITLE
stack chart changes

### PR DESCRIPTION
**Pull request by janwright73**
first draft of stack chart changes

### CODE REVIEW

File: react-charts/stack-chart/step2.md [hunk 1]
```diff
@@ -1,24 +1,49 @@
...
+  );
 }
+
+export default App;
 </pre>
 
-- Because Victory renders responsive containers, the Chart's width and height props do not determine the width and height of the chart in number of pixels, but instead define an aspect ratio for the chart. 
-The exact number of pixels will depend on the size of the container the chart is rendered into. 
-Typically, the parent container is set to the same width in order to maintain the aspect ratio.
+`ChartBar` is the React component used to add data to the Chart -- `ChartArea` could be used instead.  The `domainPadding` property adjusts the position of the first/last `ChartBar`.
```
Comment (christiemolloy) [inline react-charts/stack-chart/step2.md]:
```suggestion
`ChartBar` is the React component used to add data to the Chart, however `ChartArea` could be used instead.  The `domainPadding` property adjusts the position of the first/last `ChartBar`.
```

File: react-charts/stack-chart/step3.md [hunk 2]
```diff
@@ -1,54 +1,64 @@
...
-    &lt;/div&gt;
-  );
-}
-
-export default App;
+Add multiple datasets to the stack chart.
+
+1) <strong>Make sure `App.js` is still open.</strong>
+
+2) <strong>Locate the code within the `<Chart/>` component.</strong>
```
Comment (christiemolloy) [inline react-charts/stack-chart/step3.md]:
```suggestion
2) <strong>Locate the code within the `<ChartBar/>` component.</strong>
```

File: react-charts/stack-chart/step4.md [hunk 3]
```diff
@@ -1,65 +1,35 @@
...
-/&gt;
+&lt;Chart
+  domainPadding={{ x: [30, 25] }}
+  height={250}
+  width={600}
+&gt;
 </pre>
 
-3) Replace that section with the following code:
+3) <strong>Add a `containerComponent` property to the `Chart` component.</strong>
```
Comment (christiemolloy) [inline react-charts/stack-chart/step4.md]: Need to break this step up into multiple steps, like how its done in the Area Chart - step 4

File: react-charts/stack-chart/step5.md [hunk 4]
```diff
@@ -1,34 +1,58 @@
...
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the code for the `<Chart>` component.</strong>
 
-2) Locate the code that looks like the following:
+It should look like this:
 
 <pre class="file">
 &lt;Chart
+  containerComponent={&lt;ChartVoronoiContainer labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`} /&gt;}
```
Comment (christiemolloy) [inline react-charts/stack-chart/step5.md]: Change this so its not on 1 line:
Screen Shot 2019-12-09 at 9 23 32 AM

File: react-charts/stack-chart/step6.md [hunk 5]
```diff
@@ -1,54 +1,46 @@
...
-
-4) Locate the code that looks like the following:
+  `right: 200`
 
-<pre class="file">
-&lt;ChartStack&gt;
-...
-&lt;/ChartStack&gt;
-</pre>
+4) <strong>Add the `legendOrientation` property to the `<Chart>` component.</strong>
```
Comment (christiemolloy) [inline react-charts/stack-chart/step6.md]: Update this step to mimic how its done in the area chart tutorial:

Screen Shot 2019-12-09 at 9 26 10 AM

File: react-charts/stack-chart/step7.md [hunk 6]
```diff
@@ -1,56 +1,37 @@
...
+Change the stack chart theme color.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the `<Chart>` component and add a property to change the theme.</strong>
 
-2) Locate the code that looks like the following:
+
```
Comment (christiemolloy) [inline react-charts/stack-chart/step7.md]: As per area-chart changes, you can remove the "it should look like this" / and the code snippet.. see area-chart step 7

File: react-charts/stack-chart/step5.md (hunk 4 above)
Comment (janwright73) [reply react-charts/stack-chart/step5.md]: matches chartArea exactly now

**Pull request merged**